scripts/speed-tile-work.py

```diff
@@ -129,6 +131,23 @@ def download(src_bucket, tile_level, tile_index, week):
       speed_tiles, osmlr = convert(tile[0], tile[1], args.week, histograms)
       #move the speed tile to its destination
       upload(args.dest_bucket, tile[0], tile[1], args.week, speed_tiles)
+      # create the corresponding referencetile jobs
+      job_name = '_'.join([week.replace('/', '-'), str(t[0]), str(t[1])])
+      job = {'speed_bucket': args.src_bucket, 'ref_speed_bucket': args.reference_dest_bucket, 'tile_level': str(tile[0]), 'tile_index': str(tile[1]), 'week': week}
```

Comment:
this should be `'speed_bucket': args.dest_bucket` because it needs to get the speed tiles as input